data/names.yaml

```diff
@@ -680,7 +679,7 @@ names:
       kl: qilaliarfik
       mg: Andro niakarana
       lb: Christi Himmelfaart
-      nl: O.L.H. Hemelvaart
+      nl: Hemelvaart
```

Comment:
This one I added an override for in the Dutch holidays: https://github.com/commenthol/date-holidays/blob/161e86f0c52ecae58a7944276cae280ac39ef84e/data/countries/NL.yaml#L58

I went with `Hemelvaartsdag` as that seemed to be slightly more official, though I've certainly seen just `Hemelvaart` as well.

I kept `O.L.V. Hemelvaart` and `O.L.H. Hemelvaart` in the default translations to disambiguate them.

Reply:
thanks, fixed